src/NuGet.Core/NuGet.ProjectManagement/Projects/MSBuildNuGetProject.cs

```diff
@@ -605,6 +605,13 @@ private static bool IsAssemblyReference(string filePath)
                 throw new ArgumentNullException(nameof(context));
             }
 
+            // Some projects like website project don't have project file.
+            // Return empty list for this case.
+            if (MSBuildNuGetProjectSystem.ProjectFileFullPath == null)
```

Comment:
Is there any way this could be empty? If so use String.IsNullOrEmpty, some project systems will return string.empty instead of null for missing properties.